app/src/cards/MapCard.tsx

```diff
@@ -51,83 +54,101 @@ function MapCard(props: {
   ];
   const [race, setRace] = useState<string>(RACES[0]);
 
-  let mapData = props.data.filter((r) => r[props.varField] !== undefined);
-  if (props.fips.code !== USA_FIPS) {
-    // TODO - this doesn't consider county level data
-    mapData = mapData.filter((r) => r.state_fips === props.fips.code);
-  }
-  if (props.enableFilter) {
-    mapData = mapData.filter((r) => r.race_and_ethnicity === race);
-  }
+  const datasetStore = useDatasetStore();
+
+  const per100kVariable = per100k(props.variable) as VariableId;
+  const variableDisplayName = VARIABLE_DISPLAY_NAMES[per100kVariable];
+  const allGeosBreakdowns = Breakdowns.byState().andRace(
+    props.nonstandardizedRace
+  );
+  const allGeosQuery = new VariableQuery(per100kVariable, allGeosBreakdowns);
 
   return (
     <CardWrapper
-      datasetIds={props.datasetIds}
-      titleText={`${
-        props.varFieldDisplayName
-      } in ${props.fips.getFullDisplayName()}`}
+      queries={[allGeosQuery]}
+      datasetIds={getDependentDatasets([per100kVariable])}
+      titleText={`${variableDisplayName} in ${props.fips.getFullDisplayName()}`}
     >
-      <CardContent className={styles.SmallMarginContent}>
-        <MapBreadcrumbs
-          fips={props.fips}
-          updateFipsCallback={props.updateFipsCallback}
-        />
-      </CardContent>
+      {() => {
+        const dataset = datasetStore
+          .getVariables(allGeosQuery)
+          .filter((row) => row.race_and_ethnicity !== "Not Hispanic or Latino");
+
+        let mapData = dataset.filter((r) => r[per100kVariable] !== undefined);
```

Comment:
I'd also check for null. You can either do this explicitly with `!== undefined && !== null`, or use `!=` instead of `!==`.